balboa-http/README.md

```diff
@@ -45,3 +45,6 @@ sbt balboa-http/test
 sbt balboa-http/run "-Dbalboa.config=balboa-agent/src/main/resources/config/config.properties"
 sbt balboa-http/it:test
 
+### Load Tests
+sbt gatling-it:test
```

Comment:
How do I point this at a different server? I think it should work like this:

    SERVICE_HOST=127.0.0.1 SERVICE_PORT=2012 sbt gatling-it:test

but give it a try.